src/ProjectTemplates/Web.ProjectTemplates/WebApi-CSharp.csproj.in

```diff
@@ -9,8 +9,8 @@
   <!--#if (OrganizationalAuth || IndividualB2CAuth || EnableOpenAPI) -->
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="${MicrosoftAspNetCoreAuthenticationJwtBearerPackageVersion}" NoWarn="NU1605" />
```

Comment:
@JunTaoLuo for the `NoWarn` bit. It does not appear to be required, but regardless our project templates shouldn't suppress build warnings.